go/gas-oracle/oracle/updater_interface.go

```diff
@@ -111,8 +111,8 @@ func wrapUpdateL2GasPriceFn(backend DeployContractBackend, cfg *Config) (func(ui
 
 		// Only update the gas price when it must be changed by at least
 		// a paramaterizable amount.
-		if !isDifferenceSignificant(currentPrice.Uint64(), updatedGasPrice, cfg.significanceFactor) {
-			log.Info("gas price did not significantly change", "min-factor", cfg.significanceFactor,
+		if !isDifferenceSignificant(currentPrice.Uint64(), updatedGasPrice, cfg.l2GasPriceSignificanceFactor) {
+			log.Info("gas price did not significantly change", "min-factor", cfg.l2GasPriceSignificanceFactor,
 				"current-price", currentPrice, "next-price", updatedGasPrice)
 			txNotSignificantCounter.Inc(1)
```

Comment:
Add a ticket on linear about enabling metrics for the `gas-oracle` and then add them to the new code